src/conftest.py

```diff
@@ -28,3 +32,58 @@ def dll_initial_2():
     from dll import DoublyLinkedList
     temp = DoublyLinkedList(TEST_DLL_INIT_DATA2)
     return temp
+
+
+"""Heap Fixtures"""
+
+Heap_Fix = namedtuple("Heap_Fix", ("data",
+                                   "result",
+                                   "output",
+                                   "input",
+                                   "instance",
+                                   "expected",
+                                   "initial",
+                                   "childs",
+                                   ))
+Heap_Fix.__new__.__defaults__ = (None,) * len(Heap_Fix._fields)
+# the above line sourced from:
+# http://stackoverflow.com/questions/11351032/named-tuple-and-optional-keyword-arguments
+
+HEAP_FIX_ONE_DATA = [
+    ((1, ), 1, (None, None)),
```

Comment:
No need for all of these extra parens around every line -- they don't change anything and just make the code harder to read.